app/errors.py

```diff
@@ -0,0 +1,15 @@
+# All errors should be defined in the errors.py module.
+class VaccineError(Exception):
+    pass
```

Comment:
You should add a descriptive error message to your `VaccineError` class. This can be done by overriding the `__init__` method of the `Exception` class. For example:

python
class VaccineError(Exception):
    def __init__(self, message="Vaccine error occurred."):
        super().__init__(message)
```